action.yml

```diff
@@ -84,14 +101,14 @@ runs:
          PACKAGES: "${{ inputs.packages }}"
 
     - id: upload-logs
-      if: ${{ inputs.debug == 'true' }}
+      if: env.CACHE_KEY && inputs.debug == 'true'
       uses: actions/upload-artifact@v4
       with:
         name: cache-apt-pkgs-logs_${{ env.CACHE_KEY }}
         path: ~/cache-apt-pkgs/*.log
 
     - id: save-cache
-      if: ${{ ! steps.load-cache.outputs.cache-hit }}
+      if: env.CACHE_KEY && ! steps.load-cache.outputs.cache-hit

```

Comment:
If we decide to use this, it should use expression syntax still here and above.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs

Reply:
OK. You prefer not to omit `${{ }}`  for `if` clauses. I guess this applies to single expressions as well. I've adjusted them.